inference-engine/tests/functional/plugin/gpu/remote_blob_tests/cldnn_remote_tensor_tests.cpp

```diff
@@ -0,0 +1,361 @@
+
+    // net_local.getInputsInfo().begin()->second->setLayout(Layout::NCHW);
+    // net_local.getInputsInfo().begin()->second->setPrecision(Precision::U8);
+    // net_local.getInputsInfo().begin()->second->getPreProcess().setColorFormat(ColorFormat::NV12);
+
+    auto exec_net_b1 = ie.compile_model(fn_ptr_local, CommonTestUtils::DEVICE_GPU);
+
+    auto inf_req_local = exec_net_b1.create_infer_request();
+
+    // Run regular input for each image and compare against batched tensor
+    for (int i = 0; i < num_batch; i++) {
+        auto y_tensor = ov::runtime::Tensor{ov::element::u8, {1, 1, height, width}};
+        auto uv_tensor = ov::runtime::Tensor{ov::element::u8, {1, 2, height / 2, width / 2}};
+        auto tensor = ov::runtime::Tensor::create_nv12(y_tensor, uv_tensor);
+        inf_req_local.set_tensor(exec_net_b1.get_parameters().front()->get_friendly_name(), tensor);
+        inf_req_local.infer();
+        auto output_tensor_local = inf_req_local.get_tensor(
+            exec_net_b1.get_results().front()->input(0).get_source_output().get_node()->get_friendly_name());
+
+        // This network generates [1, size] tensor whether batch=1 or 2. So need to split
+        auto split_shared_tensor = ov::runtime::Tensor{output_tensor_local.get_element_type(),
+                                        output_tensor_local.get_shape(),
+                                        outputTensor_shared.data<float_t>() + output_tensor_local.get_size() * i};
+        ASSERT_EQ(output_tensor_local.get_size(), split_shared_tensor.get_size());
+        float thr = 0.1;
+
+        FuncTestUtils::compare_tensor(output_tensor_local, split_shared_tensor, thr, "", false);
+    }
+}
+
+const std::vector<size_t> num_batches{1, 2, 4};
+
+INSTANTIATE_TEST_SUITE_P(smoke_RemoteTensor, OVRemoteTensorBatched_Test, ::testing::ValuesIn(num_batches), OVRemoteTensorBatched_Test::getTestCaseName);
+
+using TwoNetsParams = std::tuple<size_t,   // number of streams
+                                 size_t>;  // number of requests
+
+class OVRemoteTensorTwoNets_Test : public CommonTestUtils::TestsCommon,
+    public testing::WithParamInterface<TwoNetsParams> {
+    void SetUp() override {
+        std::tie(num_streams, num_requests) = this->GetParam();
+        fn_ptrs = {ngraph::builder::subgraph::makeSplitMultiConvConcat(),
+                   ngraph::builder::subgraph::makeMultiSingleConv()};
+    };
+public:
+    static std::string getTestCaseName(const testing::TestParamInfo<TwoNetsParams>& obj) {
+        size_t streams, requests;
+        std::tie(streams, requests) = obj.param;
+        return "_num_streams_" + std::to_string(streams) + "_num_req_" +
+            std::to_string(requests);
+    }
+
+protected:
+    size_t num_streams;
+    size_t num_requests;
+    std::vector<std::shared_ptr<ngraph::Function>> fn_ptrs;
+};
+
+TEST_P(OVRemoteTensorTwoNets_Test, canInferTwoExecNets) {
```

Comment:
what does this test check?

Reply:
@vladimir-paramuzov What `TEST_P(TwoNets_Test, canInferTwoExecNets)` tests?